src/routes/admin/screenmessage/+page.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import { pb } from "$lib/stores/authStore";
+
+  export let data;
+  let screenMessages = data.screenMessages;
+  let activeMessage = data.activeMessage[0];
+
+  let open = !activeMessage.isVisible;
```

Comment:
Kunne hatt et bedre navn? At den er åpen betyr som oftest at de ikke vil vise en melding, men det betyr ikke at de to verdiene er ekvivalente, selv om de er like.

Reply:
Jeg klarte ikke å bestemme meg for `open` eller `closed`. Radio knappen for "Åpent" er `checked={open}` som ville da blitt `checked={!closed}`.

Reply:
Kan den heller hete noe direkte relatert til meldingen, som for eksempel `messageVisible`?